src/chunkserver/chunkserver-common/cmr_disk_unittest.cc

```diff
@@ -0,0 +1,127 @@
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Warray-bounds"
+#pragma GCC diagnostic ignored "-Wstringop-overflow"
+#include <gtest/gtest.h>
+#pragma GCC diagnostic pop
+#include <fstream>
+#include "chunk_trash_manager.h"
+#include "cmr_disk.h"
+#include "errors/saunafs_error_codes.h"
+#include "chunk_with_fd.h"
```

Comment:
Please group and sort the includes. Usually we:
- Include common/platform.h first.
- Include standard headers in a separate group.
- Include our files prefixed with the module for better sorting:
  chunkserver-common/chunk_trash_manager.h
  chunkserver-common/chunk_with_fd.h
  chunkserver-common/cmr_disk.h
  errors/saunafs_error_codes.h